src/runtime/lexer.c

```diff
@@ -53,6 +53,10 @@ static void ts_lexer__start(TSLexer *self, TSStateId lex_state) {
 static void ts_lexer__start_token(TSLexer *self) {
   LOG("start_token chars:%lu", self->current_position.chars);
   self->token_start_position = self->current_position;
+
+  LOG("start_token row:%lu", self->current_point.row);
+  LOG("start_token column:%lu", self->current_point.column);
```

Comment:
Could you put these in the same log message, like `start_token chars:%lu, row:%lu, column:%lu`?